### ISSUE TITLE
WP Idea Stream in WP Plugins Repository?

### ISSUE THREAD

**Issue by JeffaCubed** [imath/wp-idea-stream#69]
Hello/bonjour, This is more of a "question" than an "issue". I just wanted to confirm that WP Idea Stream is no longer available/updatable via the WP Plugins Repository (& now only available/updated here on GitHub)?

WP Plugin Page: [https://en-ca.wordpress.org/plugins/wp-idea-stream/](url)

That's it/c'est tous, Jeff (Arsenault)

**Comment by imath** [comment]
Hi Jeff,

Yes, i confirm. I removed all my plugins from there.

imath.
http://profiles.wordpress.org/imath/

**Comment by cdhinton** [comment]
Hi,
Is there a particular reason for that?
Thanks.

**Comment by imath** [comment]
Hi,

Yes i've read somewhere the WP plugins directory wasn't a place for developers to get constructive contributions or conversations. It's unfortunately true! It got worse year after year and it's now a place to download free versions of professionnal plugins who are staffed to take care of the people crying into their support forums

After 7 years being there with people rarely saying "hi" or "thanks" and regularly complaining instead of contributing, i got tired of this place. i'm not building plugins for a living so i don't want to be annoyed by these people anymore.

Not all users are like that, it's probably a minority but "good" users remain silent most of the time and this doesn't compensate frustration.

Thanks for your comprehension.

imath.
http://profiles.wordpress.org/imath/

**Comment by cdhinton** [comment]
Thanks for the quick reply, and I can understand your position. Thanks for what you're creating, especially as it's not your main line of work.

All the best.

**Comment by JeffaCubed** [comment]
Good call! I agree, (unfortunately) the WP Plugins Directory is not (or no longer) a particularly constructive nor useful place for active plugins development/conversations/etc. WP-Idea-Stream + BP-Idea-Stream are particularly impressive & I use both of these on x1 of the primary client projects - so MANY THANKS for building these! As a developer, the wp-idea-stream-custom.php file is "extremely useful" for customization. If there was a pro (paid) version of IdeaStream &/or a way to buy/rent premium support for wp-idea-stream-custom.php customizations, I would gladly buy it!! -Jeff, Vancouver, Canada

**Comment by jbostoen** [comment]
I'm glad to have found this! I actually found the plugin about a year ago after a company presented a very similar platform but way too expensive. I stumbled upon your project through the WP Plugins. 

Yesterday, the same company came back to our headquarters to present their product once again (after the whole project got on hiatus). I was also wondering why it was removed there. Good to see it's still maintained, I'll definitely give it another shot to see if it can match our needs and let management decide.

Thank you very much for your hard work! :)

If we do use it in production (or maybe if we don't), I'm planning to contribute a Dutch translation.

**Comment by imath** [comment]
@jbostoen Thanks for your feedback 😍
